client/scss/components/_button.scss

```diff
@@ -81,7 +81,7 @@
     &:hover {
       color: $color-white;
       border-color: transparent;
-      background-color: $color-button-no-hover;
+      background-color: $color-button-no;
```

Comment:
👍 I was hesitant about this because in the meantime it would mean declaring two variables which use the same value, which felt confusing. Will go with your suggestion.

Reply:
That's fair enough if you think the risk of shipping buttons without hover states is pretty low because that work will definitely get done for 4.0 :)

Reply:
Can confirm we will definitely need button styles re-implemented before the Wagtail 4 RC!